src/components/MainPane.svelte

```diff
@@ -334,7 +334,7 @@
           ? 'arrow_upward'
           : 'arrow_downward'}
         on:click={() => {
-          messageDisplay.scrollToRecent();
+          messageDisplay.scrollToRecent(true);
```

Comment:
Smooth scrolling has the same problem as when I tried to implement it a long time ago, it won't scroll all the way if new messages show up while the scroll animation is running.

Granted, I don't think this will happen often during proper usage of LTL, but it was decided previously to just scrap the idea of smooth scrolling to prevent this.

https://user-images.githubusercontent.com/20059164/162567382-4098c84e-51f6-4208-806e-672ca42f494d.mp4

Reply:
Fixed in [5ccc1ca](https://github.com/LiveTL/LiveTL/pull/408/commits/5ccc1ca8919c134aa56ff174f67e3bcef3e167b4)